website/docs/reference/administrative-commands/createIndexes.mdx

```diff
@@ -0,0 +1,125 @@
+| ------------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `createIndexes`           | string   | The name of the collection.                                                                                                                                                                                                               |
+| `indexes`                 | array    | An array of index specification documents. Each document defines an index.                                                                                                                                                                |
+| `key`                     | document | A single document that contains the field(s) to index and their sort order or type (e.g. `1` for ascending, `-1` for descending, `text` for full-text indexes, `cosmosSearch` for vector indexes, and `2dsphere` for geospatial indexes). |
+| `name`                    | string   | A required name for the index. Must be unique to the collection.                                                                                                                                                                          |
+| `unique`                  | boolean  | Optional. Ensures all values are unique for the indexed field if set to `true`.                                                                                                                                                           |
+| `partialFilterExpression` | document | Optional. Restricts index to documents matching the condition.                                                                                                                                                                            |
+| `sparse`                  | boolean  | Optional. Only indexes documents that contain the indexed field if set to `true`.                                                                                                                                                         |
+| `expireAfterSeconds`      | int32    | Optional. Specifically used for creating TTL indexes. Automatically deletes documents after set time.                                                                                                                                     |
+| `cosmosSearchOptions`     | document | Optional. Specifies the configuration parameters (`kind`, `similarity`, `dimensions`, etc.) for creating a vendor index [See vector index guide for more details](../../guides/vector-search.mdx)                                         |
```

Comment:
```suggestion
| `cosmosSearchOptions`     | document | Optional. Specifies the configuration parameters (`kind`, `similarity`, `dimensions`, etc.) for creating a vendor index. [See vector index guide for more details](../../guides/vector-search.mdx).                                         |
```
Looks not formatted
![image](https://github.com/user-attachments/assets/282d5700-fc2d-485a-8b0c-55e2823fa6c1)